packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -3536,7 +3536,7 @@ describe("IModelTransformerHub", () => {
       rootSubject: { name: masterIModelName },
     });
     // eslint-disable-next-line deprecation/deprecation
-    masterSeedDb.nativeDb.setITwinId(iTwinId); // workaround for "ContextId was not properly setup in the checkpoint" issue
+    // masterSeedDb.nativeDb.setITwinId(iTwinId); // workaround for "ContextId was not properly setup in the checkpoint" issue
```

Comment:
is this workaround no longer needed, and are we just keeping these here for future reference?

Reply:
I frankly have no idea, insufficient information provided in comment.
CI checks are not running on this PR any more because target branch was changed. Maybe some integration tests would fail?

Reply:
added the CI for the target branch. i imagine we still need this workaround, but not sure and dont want to block the pr. will create a separate issue / todo item to review this

Reply:
added to https://github.com/iTwin/imodel-transformer/issues/227